src/components/List/List.tsx

```diff
@@ -164,8 +165,9 @@ export class List<T = unknown> extends React.Component<ListProps<T>, ListState<T
                 this.handleKeyMove(event, -(activeItem || 0) - 1);
                 break;
             }
+            case 'Tab':
```

Comment:
Why does `Tab` have to select an element? The user wanted to focus on the next control and pressed Tab, but as a side effect, select the list item. IMHO, this behaviour is unexpected.